website/src/components/molecule-parser/DataInut.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react"
+import Button from "@mui/material/Button"
+import Paper from "@mui/material/Paper"
+import InputLabel from "@mui/material/InputLabel"
+import { styled } from "@mui/material/styles"
+import TextareaAutosize from "@mui/material/TextareaAutosize"
+import { BytesCodec } from "@ckb-lumos/codec/lib/base"
+import { SectionContainer } from "./SectionContainer"
+import { JSONTree } from "react-json-tree"
+import { BITranslatedUnpackType, deepStringifyNumber } from "./utils"
+
+const InputArea = styled(TextareaAutosize)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
+  ...theme.typography.body1,
+  width: "100%",
+  marginTop: "0.5rem",
+  color: theme.palette.text.secondary,
+}))
+
+type Props = {
+  codec: BytesCodec | undefined
+}
+
+const formatInput = (input: string): string => {
+  if (!input.startsWith("0x")) {
+    return "0x" + input
+  }
+  return input
+}
+
+const isBlank = (data: BITranslatedUnpackType): boolean => {
+  if (!data) {
+    return true
+  }
+  return false
+}
+
+export const DataInput: React.FC<Props> = (props) => {
+  const [inputData, setInputData] = useState<string>("")
+  const [result, setResult] = useState<BITranslatedUnpackType>(undefined)
+  const [errorMsg, setErrorMsg] = useState<string>("")
+  const handleDecode = () => {
+    if (!props.codec) {
+      setErrorMsg("please select codec")
+      return
+    }
+    try {
+      const result = props.codec.unpack(formatInput(inputData))
+      setResult(deepStringifyNumber(result))
```

Comment:
why do we still need a `deepStringifyNumber` here? the `props.codec.unpack` is not a plain JSON here?